src/Agent/NewRelic/Agent/Core/Utilization/VendorInfo.cs

```diff
@@ -155,6 +159,27 @@ public IVendorModel GetAzureFunctionVendorInfo()
             return new AzureFunctionVendorModel(appName, cloudRegion);
         }
 
+        public IVendorModel GetAzureAppServiceVendorInfo()
+        {
+            // WEBSITE_OWNER_NAME should container the subscription ID and the resource group name, separated by a '+' sign.
+            var candidateSubscriptionId = GetProcessEnvironmentVariable("WEBSITE_OWNER_NAME");
+            if (string.IsNullOrWhiteSpace(candidateSubscriptionId) || !candidateSubscriptionId.Contains('+'))
+            {
+                return null;
+            }
+
+            var subscriptionId = candidateSubscriptionId.Split('+')[0];
+            var resourceGroupName = GetProcessEnvironmentVariable("WEBSITE_RESOURCE_GROUP");
+            var siteName = GetProcessEnvironmentVariable("WEBSITE_SITE_NAME");
+
+            if (string.IsNullOrWhiteSpace(subscriptionId) || string.IsNullOrWhiteSpace(resourceGroupName) || string.IsNullOrWhiteSpace(siteName))
+            {
+                return null;
```

Comment:
Would be nice to have some logging before returning null so we can tell why we're returning null.